README.md

```diff
@@ -156,13 +186,17 @@ That's it!
 
 ## JavaScript API
 
-When installed via npm, Dart Sass supports a JavaScript API that's fully
+When installed via npm, Dart Sass supports a JavaScript API that allows users to
+compile Sass files *and* to define custom handlers for `@use` and `@import`
+rules and custom functions.
+
+
+that's fully
```

Comment:
Is the rest of this sentence still supposed to be here?

Reply:
Nope, I meant to delete this entire section.